### DISCUSSION TITLE
Welcome to interoperability-handbook Discussions!

### DISCUSSION THREAD

**Discussion by nitantdube** [ceos-org/interoperability-handbook#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nitantdube** [comment]
# Please Note a login to github account is required for posting your comments/suggestions

**Comment by yasjka** [comment]
For good interoperability, there is as well the need to coordinate catalogue metadata, which would enable independent data centers to explore data and exchange catalogue information (i.e. rather then duplicating files etc). This will enable users to explore via one data centre the global data set. Likely this fits best in the Interface section
